### PR TITLE
Port `parabricks/dbsnp` to nf-test

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -676,6 +674,7 @@ jobs:
             path: modules/nf-core/xeniumranger/resegment
           - profile: conda
             path: modules/nf-core/xeniumranger/import-segmentation
+          - path: modeules/nf-core/parabricks/dbsnp
```
Comment (famosab) [inline .github/workflows/test.yml:677]:
```suggestion
          - path: modules/nf-core/parabricks/dbsnp
```

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please :pray:

File: modules/nf-core/parabricks/dbsnp/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "parabricks"
+    tag "parabricks/dbsnp"
+    tag "gpu"
+
+    test("human - vcf") {
+
+        config './nextflow.config'
```
Comment (famosab) [inline modules/nf-core/parabricks/dbsnp/tests/main.nf.test:15]:
```suggestion
```

File: modules/nf-core/parabricks/dbsnp/main.nf [hunk 3]
```diff
@@ -1,8 +1,9 @@
 process PARABRICKS_DBSNP {
     tag "$meta.id"
     label 'process_high'
+    label 'process_gpu'
```
Comment (famosab) [inline modules/nf-core/parabricks/dbsnp/main.nf:5]:
```suggestion
    label 'process_gpu'
    stageInMode 'copy' // needed by the module to work properly - might be removed when this is fixed upstream

```

**Review by maxulysse — APPROVED** [review]

**Pull request merged**